### PR TITLE
support request count

### PR DESCRIPTION
**Pull request by wukong1992**
@YujiOshima #184 support request count: request count * request num = max suggestion num

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/193)

### CODE REVIEW

File: pkg/controller/studyjobcontroller/studyjob_controller.go [hunk 1]
```diff
@@ -477,6 +477,9 @@ func (r *ReconcileStudyJobController) checkStatus(instance *katibv1alpha1.StudyJ
 			log.Printf("Check Goal failed %v", err)
 		}
 	}
+	if instance.Status.SuggestionCount > instance.Spec.RequestCount{
+		nextSuggestionSchedule = false
```
Comment (YujiOshima) [inline pkg/controller/studyjobcontroller/studyjob_controller.go]: Not only set nextSuggestionSchedule false but mark Studyjob status as complete.
And check RequestCount only when it is larger than 0 (users set value explicitly).

Comment (wukong1992) [reply pkg/controller/studyjobcontroller/studyjob_controller.go]: @YujiOshima

Comment (YujiOshima) [reply pkg/controller/studyjobcontroller/studyjob_controller.go]: Thanks!

**Comment by wukong1992** [comment]
@gaocegege I didn't know where is wrong.

**Comment by YujiOshima** [comment]
@wukong1992 I fixed CI #194 . Please rebase master.

**Comment by YujiOshima** [comment]
@wukong1992 Thanks!

**Comment by cheyang** [comment]
@YujiOshima , would you please push new docker image wth this change? Thanks!

**Comment by YujiOshima** [comment]
@cheyang Pushed!

**Comment by cheyang** [comment]
Thank you!

**Pull request merged**